src/Configuration/Programmer/Programmer.h

```diff
@@ -43,6 +43,7 @@ enum ProgrammerErrorCode {
   BitfileNotFound = -108,
   FailedToProgramFPGA = -109,
   OpenOCDExecutableNotFound = -110,
+  FailedToProgramOTP = -111,
   InvalidFlashSize = 111,
```

Comment:
Could you help to update these error code to -112 and -113? I think this is something I missed in my previous PR.

Reply:
done.